rdkit/Chem/Draw/IPythonConsoleIntegration.py

```diff
@@ -0,0 +1,424 @@
+        confId=globals()['confId_'+uid].value
+        sconf = mol.GetConformer(confId)
+        xyz = sconf.GetPositions()
+        OwningMol = sconf.GetOwningMol()
+        
+        try:
+            labelConf=globals()['confLabel_'+uid].value
+            labelAtom=globals()['atomLabel_'+uid].value
+            if labelConf is True:
+                label = molDictKey + ':' + str(confId)
+                globals()['view_'+uid].addLabel(label, {'backgroundColor':'gray', 'fontColor':'white',
+                                                        'showBackground':'true', 'alignment':'bottomCenter'})
+            
+            if labelAtom is True:
+                label_create=[OwningMol.GetAtomWithIdx(i).GetSymbol()+
+                              str(OwningMol.GetAtomWithIdx(i).GetIdx()+1) 
+                              for i in range(sconf.GetNumAtoms())
+                             ]
+                i = None
+                for i in range(sconf.GetNumAtoms()):
+                    globals()['view_'+uid].addLabel(label_create[i], {'inFront' : 'false', 
+                                                    'fontSize' : '12',
+                                                    'fontColor':'gray',
+                                                    'showBackground':'false',
+                                                    'position' : {'x' : xyz[i][0],
+                                                                  'y' : xyz[i][1],
+                                                                  'z' : xyz[i][2]
+                                                               }
+                                                   })
+        except:
+            pass
+    
+    try:
+        bg_color_selected = globals()['background_'+uid].value
+        globals()['view_'+uid].setBackgroundColor(bg_color_selected)
+    except:
+        pass
+    
+    # To do : Add protein in viewer
+    if 'protein_'+uid in globals():
+        pdb = Chem.MolToPDBBlock(globals()['protein_'+uid], flavor=0x20|0x10)
+        globals()['view_'+uid].addModel(pdb,'pdb')
+        globals()['view_'+uid].setStyle({},{'cartoon':{'color': 'spectrum'}})
+    
+    
+    # zoomTo does not work well for surface and label... so, zoomTo should not be default settings
+    #view.zoomTo()
+    display(globals()['view_'+uid].update())
+    
+    
+def handle_change(change):
+    """This function handles all the interactive widgets except buttons and 3Dmol.js viewer"""
+    update3D(change.owner._model_id)
+    
+def handle_start_button(b):
+    """This function handles start button."""
+    b.icon='check'
+    b.description="Done!"
+    update3D(b._model_id)
+    
+def handle_zoomTo_button(b):
+    """This function handles zoomTo button"""
+    uid=globals()['rdkit_wg_dict'][b._model_id]
+    globals()['view_'+uid].zoomTo()
+    display(globals()['view_'+uid].update())
+    
+    
+    
+def ShowConformers3D(Mol, protein = None,
```

Comment:
I am actually finding a way to split it out. Probably making a class would help. Working on it.